packages/sirius-web/backend/sirius-web-infrastructure/src/main/resources/db/changelog/2024.11/01-split-representation-metadata-from-content.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2024 Obeo.
+  ~ This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v2.0
+  ~ which accompanies this distribution, and is available at
+  ~ https://www.eclipse.org/legal/epl-2.0/
+  ~
+  ~ SPDX-License-Identifier: EPL-2.0
+  ~
+  ~ Contributors:
+  ~     Obeo - initial API and implementation
+  -->
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="01-split-representation-metadata-from-content" author="gcoutable">
+        <createTable tableName="representation_content">
+            <column name="id"  type="UUID" defaultValueComputed="gen_random_uuid()">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="representation_metadata_id" type="UUID">
+                <constraints nullable="false"/>
+            </column>
+            <column name="content" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="last_migration_performed" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="migration_version" type="TEXT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created_on" type="TIMESTAMPTZ">
+                <constraints nullable="false" />
+            </column>
+            <column name="last_modified_on" type="TIMESTAMPTZ">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
+        <addForeignKeyConstraint baseTableName="representation_content" baseColumnNames="representation_metadata_id" constraintName="fk_representation_content_representation_metadata_id" referencedTableName="representation_data" referencedColumnNames="id" onDelete="CASCADE"/>
+
+        <sql>
+            INSERT INTO representation_content (
```

Comment:
Keep the id of the representation as the id for now, not to rely on gen_random_uuid()

Reply:
What does that mean "for now"?
Is `representation_metadata_id` column becoming the `Id` column, or should I have two columns with the representation id, the column `id` and the column `representation_metadata_id`?

Reply:
Done